app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -145,6 +148,43 @@ class PublicHealthHeader extends React.Component {
     );
   }
 
+  renderWarningModal() {
+    return (
+      <Modal
+        centered
+        dialogClassName="modal-import"
+        className="import-warning-modal-container"
+        scrollable="true"
+        backdrop="static"
+        size="lg"
+        show={this.state.showWarningModal}
+        onHide={async () => {
+          this.setState({ showWarningModal: false, importType: null, importData: null });
+        }}>
+        <Modal.Header closeButton>
+          <Modal.Title className="h5">{this.state.importType === 'epix' ? 'Import Epi-X' : 'Import Sara Alert Format'} Warning</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          {this.state.importData?.warnings &&
+            Object.keys(this.state.importData.warnings).map(key => (
+              <span key={key}>
+                <h5>{key}</h5>
```

Comment:
This should be a `<label>` with appropriate styling rather than an `<h5>`. H tags should only be used for page semantics rather than styling, and there is no hierarchy of H tags on this page (an h5 should only exist when its contained within h1, h2, h3, and h4 tags). Otherwise it's confusing for screen readers.
```suggestion
                <label className="h5">{key}</label>
```

Reply:
Thanks for the explanation on this requested change. See commit [8e82bce](8e82bce)